packages/plugin-akash/src/actions/getGPUPricing.ts

```
@@ -0,0 +1,225 @@
+                text: "Compare GPU prices across providers"
+            } as GetGPUPricingContent
+        } as ActionExample
+    ]],
+
+    validate: async (runtime: IAgentRuntime, message: Memory): Promise<boolean> => {
+        elizaLogger.debug("Validating GPU pricing request", { message });
+        try {
+            const params = message.content as Partial<GetGPUPricingContent>;
+
+            // Validate CPU if provided
+            if (params.cpu !== undefined && (isNaN(params.cpu) || params.cpu <= 0)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)